distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Extra_Imports.enso

```diff
@@ -26,6 +25,21 @@ polyglot java import java.math.BigInteger
 polyglot java import java.time.LocalDate
 polyglot java import java.time.LocalDateTime
 polyglot java import java.util.function.Function
+polyglot java import java.lang.Exception as JException
 polyglot java import java.lang.Thread
 polyglot java import java.lang.Thread.State
 polyglot java import java.lang.Float
+
+polyglot java import org.enso.base.Stream_Utils as JStream_Utils
+polyglot java import org.enso.base.file_system.File_Utils as JFile_Utils
+polyglot java import org.enso.base.CompareException as JCompareException
+polyglot java import org.enso.base.ObjectComparator as JObjectComparator
+
+Stream_Utils=JStream_Utils
+File_Utils=JFile_Utils
+CompareException=JCompareException
+ObjectComparator=JObjectComparator
+CaseFoldedString=JCaseFoldedString
+Text_Utils=JText_Utils
+BreakIterator=JBreakIterator
+Exception=JException
```

Comment:
Don't we have to add `## PRIVATE` comments to all of these?

We definitely do not want them to show up in GUI's Component Browser.

Reply:
The whole module is `private`.

Reply:
Oh, I missed that 🤦